sa/sa.go

```diff
@@ -2164,14 +2164,15 @@ func (ssa *SQLStorageAuthority) IncidentsForSerial(ctx context.Context, req *sap
 			return nil, err
 		}
 		if count > 0 {
-			incidentsForSerial = append(incidentsForSerial, incidentModelToPB(i))
+			incident := incidentModelToPB(i)
+			incidentsForSerial = append(incidentsForSerial, &incident)
 		}
 
 	}
 	if len(incidentsForSerial) == 0 {
 		return nil, berrors.NotFoundError("no active incidents found for serial %q", req.Serial)
 	}
```

Comment:
Is it truly an error to find no incidents for the serial? IMO it would it be better to return an empty message and no error. We should only return `berrors.NotFoundError` when we _expect_ to find something, but didn't.

Reply:
I kinda felt the same way but left it alone. Since it bothers you as well, I'll go ahead and make the appropriate change since we're a quorum.